vcpkg.json

```diff
@@ -16,12 +17,6 @@
         "zlib"
       ]
     },
-    "dds": {
-      "description": "Build support for dds files in ba2 archives",
-      "dependencies": [
-        "directxtex"
-      ]
-    },
```

Comment:
The presets file needs to reflect this feature being removed:
https://github.com/Ryan-rsm-McKenzie/bsa/blob/80f8e24b09ca35bac0b38b23ed8440d69b27ab84/CMakePresets.json#L51